src/or_trajopt/planning.py

```diff
@@ -164,7 +168,7 @@ def _addFunction(problem, timestep, i_dofs, n_dofs, fnargs):
             ValueError('Invalid cost or constraint type: {:s}'
                        .format(str(fntype)))
 
-    def _Plan(self, robot, request,
+    def _Plan(self, robot, robot_checker, request,
```

Comment:
Why do the internal methods all pass `robot_checker`, but the outer methods all use the `self` reference?